PhysicsTools/NanoAOD/plugins/VertexTableProducer.cc

```diff
@@ -131,6 +139,28 @@ void VertexTableProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSe
   pvTable->addColumnValue<float>(
       "score", pvsScoreProd.get(pvsIn.id(), 0), "main primary vertex score, i.e. sum pt2 of clustered objects", 8);
 
+  float pv_sumpt2 = 0.0;
+  for (size_t i = 0; i < (*pfcIn).size(); i++) {
```

Comment:
can you please use pfcand pointers (using auto for example) instead of index loops